### DISCUSSION TITLE
Textarea with inset label

### DISCUSSION THREAD

**Discussion by damonjentree** [tailwindlabs/tailwindcss#6107] [Ideas]
Hello!
I am using a (text) input with inset label in a project. Within the form, I also have a textarea to allow a user to enter comments. I wanted the textarea to match the text input's label. 

I have created a version of textarea that I would like to share back with the community. I have attached a screen shot of what it looks like. Since this wasn't a bug or actual feature request, I wasn't sure what the appropriate steps were.

<img width="345" alt="Screen Shot 2021-11-16 at 8 16 56 AM" src="https://user-images.githubusercontent.com/3099443/142002378-af27a004-faf0-4703-9bb4-c0eedeeab7bf.png">

**Comment by dance2die** [comment]
Nice looking element.

There is a community driven repo, **Awesome Tailwind CSS**, which you can add the link to.  
https://github.com/aniftyco/awesome-tailwindcss
